src-java/kilda-pce/src/test/java/org/openkilda/pce/impl/MaxLatencyPathComputationStrategyBaseTest.java

```diff
@@ -351,7 +350,7 @@ public void affinityOvercomeDiversity() throws Exception {
     }
 
     @Test
-    public void shouldNotFindPathGreaterThanMaxLatency() {
+    public void notFindPathGreaterThanMaxLatency() {
```

Comment:
Actually, there's an easy way to create test names: it should contain three parts: condition, action and result. E.g. <result>exception<action>InPathComputation<condition>WhenPathLatencyGreaterThanMaxLatency.
Anyway, your shorter option of the test name doesn't make the one clearer, IMHO.

Reply:
done